test/java/src/com/ibm/streamsx/topology/test/api/ParallelTest.java

```diff
@@ -43,6 +43,85 @@
 import com.ibm.streamsx.topology.tuple.BeaconTuple;
 
 public class ParallelTest extends TestTopology {
+
+    @Test
+    public void testCatsCradle() throws Exception {
+	checkUdpSupported();
+	Topology topology = new Topology("testCat");
+	TStream<String> out = topology.strings("hello").parallel(5, TStream.Routing.HASH_PARTITIONED).filter(nopStringFilter()).endParallel()
+	    .parallel(5, TStream.Routing.HASH_PARTITIONED).filter(nopStringFilter()).endParallel();
+
+	Tester tester = topology.getTester();
+	Condition<Long> expectedCount = tester.tupleCount(out, 1);
+        Condition<List<String>> hello = tester.stringContents(out, "hello");
+
+	StreamsContextFactory
```

Comment:
Can we change these to use the standard approach for writing tests, so they are submitted to the context provided by the TestTopology super class.